salt/utils/vmware.py

```diff
@@ -79,7 +80,7 @@
 import time
 
 # Import Salt Libs
-from salt.exceptions import SaltSystemExit
+import salt.exceptions as excs
```

Comment:
This should be changed to the following I think so it easier to understand the namespace wherever it gets referenced:
```
import salt.exceptions
```